composer.json

```diff
@@ -15,7 +15,9 @@
         "silverstripe/elemental-bannerblock": "^1.0",
         "silverstripe/fulltextsearch-localsolr": "dev-master",
         "silverstripe/fulltextsearch": "^3.4",
-        "benmanu/silverstripe-simple-styleguide": "^2.0"
+        "benmanu/silverstripe-simple-styleguide": "^2.0",
+        "silverstripe/tagfield": "2.1.0",
+        "silverstripe/taxonomy": "dev-master"
```

Comment:
Always advisable to use `tagged` versions of repo's  instead of branches, try bringing in `^2.0.1` instead of `dev-master` this way we can be confident that this is an official release :)